File: c/sumoftwonumbers/sumoftwonumbers2.c
#include <stdio.h>

void main()
{
        int number1;
	int number2;

	printf("First number:");
	scanf("%d",&number1);

        printf("Second number");
        scanf("%d",&number2);

	int sum = number1 + number2;

	printf("Sum of %d and %d is %d \n",number1, number2, sum);
}

File: java/sumoftwonumbers/SumOfTwoNumbers2.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

class SumOfTwoNumbers2 {
	public static void main(String args[]) throws IOException 
	{
		BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));

		System.out.println("First number:");
		int number1 = Integer.parseInt(reader.readLine());  
		System.out.println("Second number:");
		int number2 = Integer.parseInt(reader.readLine());

		System.out.println("Sum of " + number1 + " and " + number2 + " is " + (number1 + number2));
	}
}
